R/Artifact.R

```diff
@@ -46,6 +46,35 @@ ArtifactRecord <- R6::R6Class( # nolint object_name_linter
       }
     },
     #' @description
+    #' Return a backed data object. Currently only supports TileDB-SOMA
+    #' artifacts.
+    #'
+    #' @return A [tiledbsoma::SOMACollection] or [tiledbsoma::SOMAExperiment]
+    #' object
+    open = function() {
+      is_tiledbsoma <- private$get_value("suffix") == ".tiledbsoma" ||
```

Comment:
We also allow `name=="soma" `https://github.com/laminlabs/lamindb/blob/39e0f529a41d9cbc475e52fe3aa0b8095af21494/lamindb/core/storage/_backed_access.py#L84

Reply:
Ok. I'll have to see if we have access to that.

Reply:
> We also allow name=="soma"

What does this mean? And why do we allow this?

Reply:
(This is not a blocker for merging this here)